### DISCUSSION TITLE
Optional indicator on form fields

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1514] [Q&A] [answered]
Is there currently a way to indicate a field is optional? For example, in a form where most fields are mandatory and a couple are optional, we'd prefer to mark the ones that are optional instead of the required indicator on the majority of form fields. 

In Sketch, we used to be able to add the optional indicator as below. 
![image](https://user-images.githubusercontent.com/72031986/120327263-77e72800-c2e1-11eb-8f77-db8ada9c3969.png)

I don't see it in Figma or the doc site, so not sure this is built into the controls yet. Is this yet to be implemented?

**Comment by SiTaggart** [comment, marked as answer]
The optional label never actually made it into the component, so when we moved to Figma we likely didn't include it as it wasn't a true reflection if what was possible with the components in product.

It's still on our roadmap, and I believe you could do a custom composition to achieve the same thing in the mean time if you desire. It probably shouldn't get added to the Figma symbols until we've added it to the component itself unfortunately.

**Reply by PoojMenon** [reply]
Thanks Simon!
